### PR TITLE
Recipe box page for editing recipes and stuff

### PR DESCRIPTION
**Pull request by mfernandez-rally**
Recipe box where you can (eventually) add recipes in more detail and delete them forever 😲

### CODE REVIEW

File: src/app/components/recipeBox/recipeBoxComponent.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    setSelectedRecipeId: PropTypes.func.isRequired
+  };
+
+  getRecipes = () => {
+    const recipeArray = [];
+    for (var recipeId of Object.keys(this.props.recipes)) {
+      const recipe = this.props.recipes[recipeId];
+      recipeArray.push(
+        <RecipeCard className={recipeId === this.props.selectedRecipeId ? 'selectedRecipe' : ''} 
+                    {...this.props} recipeId={recipeId}
```
Comment (WadePeterson) [inline src/app/components/recipeBox/recipeBoxComponent.js]: is `{...this.props}` here just to pass in `setSelectedRecipeId`? Or were you thinking you'd want to pass more in later?

File: src/app/components/recipeBox/recipeBoxComponent.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export default class RecipeBox extends React.Component {
+  static propTypes = {
+    recipes: PropTypes.object.isRequired,
+    selectedRecipeId: PropTypes.string.isRequired,
+    setSelectedRecipeId: PropTypes.func.isRequired
+  };
+
+  getRecipes = () => {
+    const recipeArray = [];
+    for (var recipeId of Object.keys(this.props.recipes)) {
```
Comment (WadePeterson) [inline src/app/components/recipeBox/recipeBoxComponent.js]: Could change this to use `map` if you wanted:

```
return Object.keys(this.props.recipes).map(recipeId => {
  const recipe = this.props.recipes[recipeId];
  return <RecipeCard ... />;
});
```

**Comment by WadePeterson** [comment]
🔥 🥊

File: src/app/components/recipeBox/recipeBoxComponent.js (hunk 1 above)
Comment (mfernandez-rally) [inline src/app/components/recipeBox/recipeBoxComponent.js]: Also to pass `addRecipe` and `deleteRecipe` even though they aren't getting used yet

**Pull request merged**